content/actions/reference/events-that-trigger-workflows.md

```diff
@@ -677,6 +677,12 @@ on:
     types: [published]
 ```
 
+{% note %}
+
+**Note:** The `prereleased` type will not trigger for pre releases getting published from draft releases. The `published` type however will. If you want a workflow to run when stable *and* pre releases are published, subscribe to `published` instead of `released` and `prereleased`.
```

Comment:
```suggestion
**Note:** The `prereleased` type will not trigger for pre-releases published from draft releases, but the `published` type will trigger. If you want a workflow to run when stable *and* pre-releases publish, subscribe to `published` instead of `released` and `prereleased`.
```